curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/65ae458e23954c3469e0c209.md

```diff
@@ -0,0 +1,112 @@
+---
+id: 65ae458e23954c3469e0c209
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+
+# --description--
+
+In earlier projects you learned about the `setAttribute` method. Another way to update an attribute in JavaScript is to use the following syntax:
+
+```js
+el.attribute = value;
+```
+
+The property names for hyphenated HTML attribute values, such as `aria-label`, follow camel case, becoming `ariaLabel`.
+
+```js
+el.ariaLabel = "Aria Label Value";
+```
+
+Set the `aria-label` attribute for the `input` element to the same value as the `id` attribute.
+
+
+# --hints--
+
+You should set an attribute using the syntax `el.attribute = value`.
+
+```js
+assert.match(code, /input\.aria[lL]abel\s*=/);
+
+```
+
+Property names should follow camel case when updating hyphenated HTML attribute values.
+
+```js
+assert.match(code, /input\.ariaLabel\s*=/);
+
+```
```

Comment:
I don't think we need this test.
We can just use the first one with some updates.

```suggestion
```